src/Generators/MigrationGenerator.php

```diff
@@ -194,7 +194,7 @@ protected function buildDefinition(Model $model): string
                 );
             }
 
-            if (!empty($columnAttributes) && !$this->isIdOrUuid($column->dataType())) {
+            if (!empty($columnAttributes) && !$this->isIdUlidOrUuid($column->dataType())) {
```

Comment:
This method is getting a bit silly, please refactor it to `isIdColumnType`

Reply:
Agreed :)

Reply:
Ok. Please change it.